From b3528ac97ab6b8f33179ed706f6fb09acf28d06a Mon Sep 17 00:00:00 2001
From: Amos Kong <akong@redhat.com>
Date: Tue, 14 Jan 2014 09:56:40 +0100
Subject: [PATCH 16/16] vl: make boot_strict variable static (not used outside vl.c)

RH-Author: Amos Kong <akong@redhat.com>
Message-id: <1389693400-21546-3-git-send-email-akong@redhat.com>
Patchwork-id: 56653
O-Subject: [RHEL-7.0 qemu-kvm PATCH 2/2] vl: make boot_strict variable static (not used outside vl.c)
Bugzilla: 997817
RH-Acked-by: Laszlo Ersek <lersek@redhat.com>
RH-Acked-by: Paolo Bonzini <pbonzini@redhat.com>
RH-Acked-by: Markus Armbruster <armbru@redhat.com>

From: Michael Tokarev <mjt@tls.msk.ru>

Signed-off-by: Michael Tokarev <mjt@tls.msk.ru>
Reviewed-by: Amos Kong <akong@redhat.com>
(cherry picked from commit 6f2bfda35cf1c7b8a5913570b808396ab9299873)
---
 vl.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

Signed-off-by: Miroslav Rezanina <mrezanin@redhat.com>
---
 vl.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/vl.c b/vl.c
index 11899dc..5aea00c 100644
--- a/vl.c
+++ b/vl.c
@@ -232,7 +232,7 @@ int ctrl_grab = 0;
 unsigned int nb_prom_envs = 0;
 const char *prom_envs[MAX_PROM_ENVS];
 int boot_menu;
-bool boot_strict;
+static bool boot_strict;
 uint8_t *boot_splash_filedata;
 size_t boot_splash_filedata_size;
 uint8_t qemu_extra_params_fw[2];
-- 
1.7.1

